CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+* ``bokulich2013-id``: This only applies to the founder datasets included in mockrobiota, and indicates the mock community ID used in the [original citation](doi:10.1038/nmeth.2276). New mock communities should list ``NA`` as the value for this field.
+* ``bokulich2015-id``: This only applies to the founder datasets included in mockrobiota, and indicates the mock community ID used in the [original citation](https://dx.doi.org/10.7287/peerj.preprints.934v2). New mock communities should list ``NA`` as the value for this field.
+* ``target-gene``: The marker gene analyzed in this mock community dataset, for example, 16S, 18S, ITS.
+* ``target-subfragment``: The subregion of the marker gene analyzed. E.g., V4 (a subregion of 16S rRNA).
+* ``study-type``: The type of study. Should be ``marker-gene`` or ``metagenome``. Other analysis types are theoretically possible. If submitting a mock community that is neither marker-gene sequences nor shotgun metagnome sequences, submit an [issue request](https://github.com/caporaso-lab/mockrobiota/issues) to confer with the mockrobiota developers before proceeding.
+* ``sequencing-instrument``: The sequencing method used for analysis, for example, ``illumina-hiseq``. We do not have an established naming convention here, but may make suggestions to help keep these consistent across data sets.
+* ``physical-specimen-available``: Does a physical specimen exist for this mock community and is it available for other investigators to request aliquots? Value should be ``Yes`` or ``No``.
+* ``physical-specimen-contact``: If the value for the ``physical-specimen-available`` is ``Yes``, this should be a contact email to which other investigators should direct their requests. Thank you for sharing your precious mock community resources with the mockrobiota community!
+
+### ``sample-metadata.tsv``
+This file lists metadata for each individual sample contained in a mock community dataset, e.g., replicates of the same mock community or other mock communities included in the same sequencing run. This is a tab-separated text file, in [QIIME 1 mapping file format](http://qiime.org/documentation/file_formats.html#metadata-mapping-files). The [keemei Google Sheets](keemei.qiime.org) plugin can be used to validate this file. The minimum requirements for microbiome and metagenome mock communities are:
```

Comment:
> Should this be The minimum requirements for marker-gene and metagenome mock communities are:

See [my question here](https://github.com/caporaso-lab/mockrobiota/pull/20/files#r63619521) to solve this generally across the document